test-types/copy.js

```diff
@@ -0,0 +1,76 @@
+const fs = require('fs')
```

Comment:
Should we move this file into a `/scripts` folder maybe (just to stay consistent with the main repo)?

Reply:
do you think it worth creating `tests` folder as well and then moving `test` and `test-types` inside?
```
tests
 - unit (or just test?)
 - types
```

Reply:
copy.js is in tests folder in the main repo - https://github.com/webdriverio/webdriverio/blob/master/tests/typings/copy.js

Reply:
> do you think it worth creating tests folder as well and then moving test and test-types inside?

Yes!

> copy.js is in tests folder in the main repo

Ooops 🙈 nevermind then